src/wp-admin/includes/upgrade.php

```diff
@@ -2293,6 +2297,28 @@ function upgrade_600() {
 	}
 }
 
+/**
+ * Executes changes made in WordPress 6.3.0.
+ *
+ * @ignore
+ * @since 6.3.0
+ *
+ * @global int $wp_current_db_version The old (current) database version.
+ */
+function upgrade_630() {
+	global $wp_current_db_version;
+
+	if ( $wp_current_db_version < 55752 ) {
+		if ( ! is_multisite() ) {
+			// Replace non-autoload option can_compress_scripts with autoload option, see #55270
+			$can_compress_scripts = get_option( 'can_compress_scripts', false );
+			if ( false !== $can_compress_scripts ) {
+				update_option( 'can_compress_scripts', $can_compress_scripts, 'yes' );
+			}
+		}
+	}
+}
```

Comment:
```suggestion
/**
 * Executes changes made in WordPress 6.3.0.
 *
 * @ignore
 * @since 6.3.0
 */
function upgrade_630() {
	if ( ! is_multisite() ) {
		// Replace non-autoload option can_compress_scripts with autoload option, see #55270
		$can_compress_scripts = get_option( 'can_compress_scripts', false );
		if ( false !== $can_compress_scripts ) {
			update_option( 'can_compress_scripts', $can_compress_scripts, 'yes' );
		}
	}
}
```

The `upgrade_630` function should only be executed when the `$wp_current_db_version` is less than 55752, so there's no need to check it here.

Reply:
@mukeshpanchal27 Compare this to other functions. https://github.com/WordPress/wordpress-develop/blob/a10e2bcf05309b814d7aba32281d24fe30eaf65b/src/wp-admin/includes/upgrade.php#L2266-L2278

This is how these work, so I going to leave this.

Reply:
Yes but i think it wroth to check same condition again in function 🤔

Reply:
> Yes but i think it wroth to check same condition again in function

Can you explain this?

Reply:
```
if ( $wp_current_db_version < 55752 ) {
	upgrade_630();
}
```

The `upgrade_630()` only call if db version is less then 55752 `if ( $wp_current_db_version < 55752 ) {` right?

```
function upgrade_630() {
	global $wp_current_db_version;
	if ( $wp_current_db_version < 55752 ) {
......
```

Again in `upgrade_630()` we check same condition `if ( $wp_current_db_version < 55752 ) {`.

Reply:
That is what the other upgrade functions do. I don't see why this one would be any different.

Reply:
I agree with @spacedmonkey here, while it seems indeed pointless, that's what all other WP core upgrade functions do, have the check both around the function and in the function. There may be some reason for that which we're missing, so I'd rather be safe and stick to that practice.

Reply:
Make sense. Thanks!